### PR TITLE
181 update deployment instructions + settings_local.template.py

### LINKED ISSUE #181: Update deployment instructions
- [x] variable AUTHENTICATION_BACKENDS needs to be explained
- [x] git clone should use HTTPS instead of SSH
- [x] advise user to change the database password
- [x] separate Solr mirror troubleshooting from assignment of ownership (chown commands)
- [x] line 460 - change directory so the command works

related to #151, #179

### PR DESCRIPTION
**Pull request by vildead**
Closes #181 #151 #179

I have some questions:
1. `ldap.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, ldap.OPT_X_TLS_NEVER)` is not included in settings template. Should it be there? I am actually not sure what that does exactly.
2. Could you please help me with the explanation of `AUTHENTICATION_BACKENDS` variable? According to #181 it should be included in deployement instructions. Should other authentication backends be included/described.?

### CODE REVIEW

**Comment by jLebioda** [comment]
@vildead FYI:
1. I'd assume that it should be there. I think it is relevant only for local deployment that uses LDAP. This very line says that TLS (maybe even TLS and SSL) should not be required when connecting to LDAP server.
2. According to the docs, it is not really crucial to tune it - i.e. the two lines in AUTHENTICATION_BACKENDS are required for the application to run; nothing to add or remove is required from the user

File: elixir_daisy/settings_local.template.py [hunk 1]
```diff
@@ -6,9 +6,24 @@
     'guardian.backends.ObjectPermissionBackend',
 ]
 
+# SECURITY WARNING: change the secret key used in production and keep it secret !
+SECRET_KEY = 'qe1lmt43v1n%66vibs0&0s9qw7i!xjs^!i#f#$t_7-r8n&=+sp'
```
Comment (jLebioda) [inline elixir_daisy/settings_local.template.py:10]: I would be slightly more in favour of:

```
L09:  # SECURITY WARNING: change the secret key used in production and keep it secret !
L10:  SECRET_KEY = None  # Generate a secret key by e.g. django.core.management.utils.get_random_secret_key()
L11:  if SECRET_KEY is None: rase NotImplementedError('You must specify SECRET_KEY in settings_local.py')
```

**Review by jLebioda — CHANGES_REQUESTED** [review]
@vildead shall we continue with a merge?

File: elixir_daisy/settings_local.template.py (hunk 1 above)
Comment (jLebioda) [reply elixir_daisy/settings_local.template.py:10]: This way:
a) if the user downloads the code, tries to run it and does nothing - he will see an error with a direct message
b) if the user changes the value, the exception will not be raised

Comment (vildead) [reply elixir_daisy/settings_local.template.py]: that looks good! :) I have added the change

**Comment by vildead** [comment]
@jLebioda I think we are good to go. The build is failing though

**Review by jLebioda — APPROVED** [review]
Looks good!
Thanks

**Comment by vildead** [comment]
Closes #151 
Closes #179

**Pull request closed without merging**